package-lock.json

```diff
@@ -1947,7 +1947,7 @@
         "source-map": {
           "version": "0.6.1",
           "resolved": "https://registry.npmjs.org/source-map/-/source-map-0.6.1.tgz",
-          "integrity": "sha512-UjgapumWlbMhkBgzT7Ykc5YXUT46F0iKu8SGXq0bcwP5dz/h0Plj6enJqjz1Zbq2l5WaqYnrVbwWOWMyF3F47g==",
+          "integrity": "sha1-dHIq8y6WFOnCh6jQu95IteLxomM=",
```

Comment:
Seems like the `package-lock.json` wouldn't need to change for this update?...

Reply:
Seems like it, but I always commit changes to lockfiles. I assume this was the result of rebuilding the docker image, but I could remove it